openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -113,6 +113,7 @@ class FormDescription(object):
         forms.Textarea: "textarea",
         forms.BooleanField: "checkbox",
         forms.EmailField: "email",
+        forms.models.ModelMultipleChoiceField: "select",
```

Comment:
I don't think "**select**" is the correct map for `ModelMultipleChoiceField` field.

`ModelMultipleChoiceField` is used when we want user to select multiple values from the dropdown. The default widget for it is [Select Multiple](https://docs.djangoproject.com/en/3.1/ref/forms/widgets/#django.forms.SelectMultiple) `<select multiple></select>`. However we do not yet have support for it, see the `ALLOWED_TYPES` list here:
https://github.com/edx/edx-platform/blob/9ecbe15d9123690319409fe04e5579e8a53e622d/openedx/core/djangoapps/user_api/helpers.py#L99
Adding this to **FIELD_TYPE_MAP** would require addition of "**_select multiple_**" to the allowed types list and how this type will be handled on frontend.

If the intended use is to just select one value, I suggest you use `ModelChoiceField`:
```suggestion
        forms.models.ModelChoiceField: "select",
```

If you also want to add support for `ModelMultipleChoiceField`, kindly make the requested changes to frontend and FormDescription to handle this field.

Reply:
@melihcolpan ^